pwndbg/color/memory.py

```diff
@@ -18,19 +24,19 @@
 )
 
 
-def get(address, text=None, prefix=None) -> str:
+def get(address: int | gdb.Value, text: str | None = None, prefix: str | None = None) -> str:
     """
     Returns a colorized string representing the provided address.
 
     Arguments:
-        address(int): Address to look up
-        text(str): Optional text to use in place of the address
-              in the return value string.
-        prefix(str): Optional text to set at beginning in the return value string.
+        address(int | gdb.Value): Address to look up
+        text(str | None): Optional text to use in place of the address in the return value string.
+        prefix(str | None): Optional text to set at beginning in the return value string.
     """
     address = int(address)
+    page = pwndbg.gdblib.vmmap.find(address)
 
-    page = pwndbg.gdblib.vmmap.find(int(address))
+    color: ColorFunction = normal
```

Comment:
Fwiw u can annotate variable without setting its value.

However, the annotation will still execute some code on each call of this function, so its entirely free, that's not that good news heh.

<img width="486" alt="image" src="https://github.com/pwndbg/pwndbg/assets/10009354/3a0a1311-b839-4a6f-8932-a7646bc5e9ce">

Reply:
`color: ColorFunction = normal` is added before so that type-checking tools can know what is the type of `color`